client/js/models/archive.js

```diff
@@ -276,9 +241,7 @@ module.exports = {
         promises.push(promise)
       } else { // download entire archive as zip
         zipName = state.key
-        var keys = Object.keys(state.entries.sort())
-        keys.forEach((key, i) => {
-          const entry = state.entries[key]
+        state.entries.forEach((entry) => {
```

Comment:
didn't @clkao have a reason he was needing to `.sort()` these for jszip? i can't remember what the reason was tho.